diff --git a/include/mariadb_python.h b/include/mariadb_python.h
index bd46c23..5b93d11 100644
--- a/include/mariadb_python.h
+++ b/include/mariadb_python.h
@@ -17,6 +17,7 @@
    51 Franklin St., Fifth Floor, Boston, MA 02110, USA
 ******************************************************************************/
 #define PY_SSIZE_T_CLEAN
+#define _POSIX_C_SOURCE 199309L
 
 #include "Python.h"
 #include "bytesobject.h"
