diff --git a/include/mariadb_python.h b/include/mariadb_python.h
index bd46c23..e433ea9 100644
--- a/include/mariadb_python.h
+++ b/include/mariadb_python.h
@@ -27,10 +27,12 @@
 #include <mysql.h>
 #include <errmsg.h>
 #include <mysqld_error.h>
-#include <time.h>
 #include <docs/common.h>
 #include <limits.h>
 
+#define _POSIX_C_SOURCE 199309L
+#include <time.h>
+
 #if defined(_WIN32)
 #include <windows.h>
 typedef CRITICAL_SECTION pthread_mutex_t;
