diff --git a/mariadb/mariadb_connection.c b/mariadb/mariadb_connection.c
index 07321b2..3311c63 100644
--- a/mariadb/mariadb_connection.c
+++ b/mariadb/mariadb_connection.c
@@ -16,6 +16,7 @@
   or write to the Free Software Foundation, Inc.,
   51 Franklin St., Fifth Floor, Boston, MA 02110, USA
  *****************************************************************************/
+#define _POSIX_C_SOURCE 199309L
 
 #include "mariadb_python.h"
 #include "docs/connection.h"
